packages/push-platform-notifications-controller/LICENSE

```diff
@@ -0,0 +1,20 @@
+MIT License
+
+Copyright (c) 2018 MetaMask
```

Comment:
Do we want to update the license year?

``` suggestion
Copyright (c) 2024 MetaMask

Reply:
LOL, correct me if wrong @Jonathansoufer. Were these scaffolded through the package script?

Reply:
Fixed.